docstring_parser/parser/google.py

```diff
@@ -115,7 +117,7 @@ def parse(text: str) -> Docstring:
         indent = indent_match.group()
 
         # Check for returns/yeilds (only one element)
-        if _sections[title] in ("returns", "yields"):
+        if _sections[title] in ("returns", "yields", "examples"):
```

Comment:
The comment above is false. Maybe it should be changed to `Check for singular elements` or something?